scipy/stats/tests/test_distributions.py

```diff
@@ -37,6 +37,10 @@
 # python -OO strips docstrings
 DOCSTRINGS_STRIPPED = sys.flags.optimize > 1
 
+# Failing on macOS 11, Intel CPUs. See gh-14901
+MACOS_INTEL = (sys.platform == 'darwin') and (platform.machine() == 'x86_64')
```

Comment:
I think we need to add `import platform` above--let me see if I can push that in real quick